stories/dropdown/index.stories.tsx

```diff
@@ -27,7 +27,7 @@ const options = [
   { value: 1, label: 'First option' },
   { value: 2, label: 'Second option' },
   { value: '3', label: 'Third option' },
-  { value: '4', label: 'Fourth option' },
```

Comment:
This was me being sloppy at cleaning up. I made the list much much longer for showing to Adam so I could give a better example view of what the list would look like with 50 options and then forgot that we have 5 initially rather than four. I'll restore the fourth option.